common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -523,7 +536,30 @@ public static void beginInteractive(final InteractiveTokenCommand command) {
                 );
             }
 
-            sInteractiveExecutor.execute(new Runnable() {
+            // It is possible that the current task could be stuck irretrievably.  If that's
```

Comment:
Do we know when this actually happens? Repro steps?

Reply:
When you create a bug in a command (that's how I got it).  You could write a command that does nothing but a thread.sleep, and that would do it.  The idea is that we shouldn't make the user close the software to recover, since we're trying to kill the running command anyway.

Reply:
Thing is that we're not be resilient to bugs in downstream software.